inc/class-thirdparties.php

```diff
@@ -0,0 +1,407 @@
+<?php
```

Comment:
I think this should be refactored to be one plugin = one compatibility class. On the long run having everything packed in the same class will bring bloat and complexity that can be avoidable.

It could also potentially allow to use an interface, and abstract class, since there is a lot of repetition in the methods